assets/js/logging.js

```diff
@@ -0,0 +1,21 @@
+var colors = require('colors');
```

Comment:
Happy to change the name, but assets is being used to store unpacked frontend assets.